DroneInventorySystem/src/main/java/com/digitalojt/web/consts/SearchParams.java

```diff
@@ -1,13 +1,13 @@
-package com.digitalojt.web.consts;
-
-/**
- * 検索条件定数クラス
- *
- * @author haruka matano
- * 
- */
-public class SearchParams {
-
-	// 最大文字数
-	public static final int MAX_LENGTH = 20;
-}
+package com.digitalojt.web.consts;
+
+/**
+ * 検索条件定数クラス
+ *
+ * @author haruka matano
+ * 
+ */
+public class SearchParams {
+
+	// 最大文字数
+	public static final int MAX_LENGTH = 15;

```

Comment:
```suggestion
	public static final int MAX_CHARACTER_LENGTH = 15;

```
何の長さかわかるように定義した方が良いでしょう。
元々がMAX_LENGTHだったかもですが。

Reply:
検索に使用する最大文字数の為名前を変更しました。

Reply:
Goodです！